src/java/org/apache/cassandra/exceptions/InvalidColumnTypeException.java

```diff
@@ -54,58 +55,9 @@ private static String msg(ByteBuffer name,
                               String reason)
     {
         return String.format("Invalid type %s for column %s: %s",
-                             invalidType.asCQL3Type(),
+                             invalidType.asCQL3Type().toSchemaString(),
                              ColumnIdentifier.toCQLString(name),
                              reason);
     }
 
-    /**
-     * Attempts to return a "fixed" (and thus valid) version of the type. Doing is so is only possible in restrained
-     * case where we know why the type is invalid and are confident we know what it should be.
-     *
-     * @return if we know how to auto-magically fix the invalid type that triggered this exception, the hopefully
-     * fixed version of said type. Otherwise, {@code null}.
-     */
-    public AbstractType<?> tryFix()
```

Comment:
Removing this method leaves all the attributes in this class unused. If we remove then and don't provide them to the constructor, we can make some simplifications in `AbstractType#columnException` and `AbstractType#validateForColumn`: https://github.com/adelapena/cassandra/commit/bac933d087ae088e8a66cbbd1be39ecddc8db6a4